modules/ROOT/partials/misc/advcode-shortcuts.adoc

```diff
@@ -1,13 +1,30 @@
-== Advanced Code Editor search and replace keyboard shortcuts
+== Enhanced Code Editor search and replace keyboard shortcuts
 
-Advanced Code Editor provides the following shortcuts within the code dialog.
+Enhanced Code Editor provides the following shortcuts within the code dialog.
 
 [cols=",,",options="header"]
 |===
-|Action |PC |Mac
-|Find |Ctrl+F |Command+F
-|Find next instance |Ctrl+G |Command+G
-|Find previous instance |Shift+Ctrl+G |Shift+Command+G
-|Replace |Ctrl+H |Command+Option+F
-|Replace All |Shift+Ctrl+R |Shift+Command+Option+F
+|Action |PC |Mac 
+|Find |Ctrl+F |Command+F 
+|Find next instance |Ctrl+G |Command+G 
+|Find previous instance |Shift+Ctrl+G |Shift+Command+G 
+|Replace |Ctrl+H |Command+Option+F 
+|Replace All |Shift+Ctrl+R |Shift+Command+Option+F 
+|Open Search Panel |Ctrl+F |Command+F 
+|Find Next |F3 |F3 
+|Find Previous |Shift+F3 |Shift+F3 
+|Close Search Panel |Esc |Esc 
+|Select Selection Matches |Ctrl+Shift+L |Command+Shift+L 
+|Select Next Occurrence |Ctrl+D |Command+D 
+|Select Next Occurrence (Atom) |Ctrl+B |Command+B 
+|Fold Code |Ctrl+Q |Command+Q+Fn
+|Accept Completion |Tab |Tab 
 |===
+
+By default {productname} also imports the below:
+
+* https://codemirror.net/docs/ref/#search:~:text=macOS)%3A%20deleteLineBoundaryForward.-,defaultKeymap,-%3A%20readonly%C2%A0[defaultKeymap]
+* https://codemirror.net/docs/ref/#search:~:text=influence%20that%20logic.-,historyKeymap,-%3A%20readonly%C2%A0[historyKeymap]
+* https://codemirror.net/docs/ref/#search:~:text=over%20that%20bracket.-,closeBracketsKeymap,-%3A%20readonly%C2%A0[closeBracketsKeymap]
+* https://codemirror.net/docs/ref/#search:~:text=all%20folded%20code.-,foldKeymap,-%3A%20readonly%C2%A0[foldKeymap]
+* https://codemirror.net/docs/ref/#search:~:text=the%20given%20amount.-,completionKeymap,-%3A%20readonly%C2%A0[completionKeymap]
```

Comment:
I'm not sure if we should be linking to external documentation here.
* [ ] Check that this is suitable

Reply:
Removed
```suggestion
```